assets/js/components/data.js

```diff
@@ -648,37 +648,15 @@ const dataAPI = {
 	},
 
 	/**
-	 * Gets module data.
+	 * Sets a module to activated or deactivated using the REST API.
 	 *
-	 * @param {string} module The data item identifier.
-	 * @param {string} datapoint The data point to retrieve. Optional, otherwise returns all data.
-	 *
-	 * @return {Promise} A promise for the fetch request.
-	 */
-	getModuleData( module, datapoint ) {
-		return this.get( 'modules', module, datapoint );
-	},
```

Comment:
I like the concept of these wrapper methods to cut down on repetitive arguments, but I think it may be better in another place (like a base Module class).

Reply:
The naming of this function (and the other similar one I renamed) is confusing though. Based on the API routes, "module data" is things that comes from a datapoint. The other function I renamed however deals with the module itself (activate/deactivate it). I'm open to introducing wrappers to simplify in the future, but only when we have a proper API figured out.